service/frontend/workflow_handler_test.go

```diff
@@ -445,8 +446,9 @@ func (s *workflowHandlerSuite) TestStartWorkflowExecution_Failed_InvalidExecutio
 		RequestId: uuid.New(),
 	}
 	_, err := wh.StartWorkflowExecution(context.Background(), startWorkflowExecutionRequest)
-	s.Error(err)
-	s.Equal(errInvalidWorkflowExecutionTimeoutSeconds, err)
+	var invalidArg *serviceerror.InvalidArgument
+	s.ErrorAs(err, &invalidArg)
```

Comment:
so you need not `s.ErrorAs` here but `s.Equal` because this is a contract between handler and interceptor which converts it ton gRPC status code.